src/mscorlib/corefx/System/Globalization/TextInfo.Unix.cs.orig

```diff
@@ -0,0 +1,99 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Diagnostics.Contracts;
+using System.Text;
+
+namespace System.Globalization
+{
+    public partial class TextInfo
+    {
+        private readonly bool m_needsTurkishCasing;
+
+        //////////////////////////////////////////////////////////////////////////
+        ////
+        ////  TextInfo Constructors
+        ////
+        ////  Implements CultureInfo.TextInfo.
+        ////
+        //////////////////////////////////////////////////////////////////////////
+        internal unsafe TextInfo(CultureData cultureData)
+        {
+            m_cultureData = cultureData;
+            m_cultureName = m_cultureData.CultureName;
+            m_textInfoName = m_cultureData.STEXTINFO;
+            m_needsTurkishCasing = NeedsTurkishCasing(m_textInfoName);
+        }
+
+        [System.Security.SecuritySafeCritical]
+        private unsafe string ChangeCase(string s, bool toUpper)
+        {
+            Contract.Assert(s != null);
+
+            if (s.Length == 0)
+            {
+                return string.Empty;
+            }
+
+            string result = string.FastAllocateString(s.Length);
+<<<<<<< HEAD
```

Comment:
Deleted. I wasn't aware of the difftool leaving *.orig files everywhere. Did a scan and removed them all as well as any other merge remnants.